src/Core/Resolvers/CosmosQueryEngine.cs

```diff
@@ -390,5 +388,15 @@ private async Task<(string? idValue, string? partitionKeyValue)> GetIdAndPartiti
             byte[] base64EncodedBytes = Convert.FromBase64String(base64EncodedData);
             return Encoding.UTF8.GetString(base64EncodedBytes);
         }
+
+        private string GetValidatedDataSourceName(string dataSourceName)
```

Comment:
why is this named "GetValidated" versus "GetDefault", there isn't any validation occurring.

Reply:
Method no longer needed. removing.